app/src/main/java/com/remindme/app/ui/component/CustomViews.kt

```diff
@@ -52,133 +49,172 @@ import com.remindme.app.ui.theme.Grey700
-  var text by remember { mutableStateOf("") }
-  Spacer(modifier = Modifier.size(10.dp))
+    var text by remember { mutableStateOf("") }
+    Spacer(modifier = Modifier.size(10.dp))
 
-  val trailingIconView = @Composable {
-    IconButton(
-      onClick = {
-        // TODO open default data picker
-      },
-    ) {
-      Image(
-        painter = painterResource(id = R.drawable.ic_calendar),
-        null
-      )
+    val trailingIconView = @Composable {
+        IconButton(
+            onClick = {
+                // TODO open default data picker
+            },
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.ic_calendar),
+                null
+            )
+        }
     }
-  }
 
-  OutlinedTextField(
-    value = text,
-    onValueChange = { text = it },
-    singleLine = true,
-    maxLines = 1,
-    readOnly = true,
-    label = {
-      Row {
-        Text(label, color = Grey700)
-        Box(
-          modifier = Modifier
-            .fillMaxWidth(),
-          contentAlignment = Alignment.BottomEnd
-        ) { Text("03-07-2022", color = Grey700) }
-      }
-    },
-    colors = TextFieldDefaults.outlinedTextFieldColors(
-      focusedBorderColor = MaterialTheme.colors.primary,
-      unfocusedBorderColor = MaterialTheme.colors.primary,
-      focusedLabelColor = MaterialTheme.colors.primary,
-      cursorColor = MaterialTheme.colors.primary,
-      textColor = MaterialTheme.colors.primary
-    ),
-    modifier = Modifier.fillMaxWidth(),
-    trailingIcon = trailingIconView
-  )
+    OutlinedTextField(
+        value = text,
+        onValueChange = { text = it },
+        singleLine = true,
+        maxLines = 1,
+        readOnly = true,
+        label = {
+            Row {
+                Text(label, color = Grey700)
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    contentAlignment = Alignment.BottomEnd
+                ) { Text("03-07-2022", color = Grey700) }
+            }
+        },
+        colors = TextFieldDefaults.outlinedTextFieldColors(
+            focusedBorderColor = MaterialTheme.colors.primary,
+            unfocusedBorderColor = MaterialTheme.colors.primary,
+            focusedLabelColor = MaterialTheme.colors.primary,
+            cursorColor = MaterialTheme.colors.primary,
+            textColor = MaterialTheme.colors.primary
+        ),
+        modifier = Modifier.fillMaxWidth(),
+        trailingIcon = trailingIconView
+    )
 }
 
 @Composable
-fun RemindBeforeCheckboxGroup() {
-  Column(modifier = Modifier.padding(top = 20.dp)) {
-    Row {
-      CheckboxWithLabel(label = "60 days")
-      Spacer(Modifier.size(width = 50.dp, height = 0.dp))
-      CheckboxWithLabel(label = "7 days")
+fun RemindBeforeCheckboxGroup(modifier: Modifier = Modifier) {
+    Column(modifier = modifier.padding(top = 10.dp)) {
+        Row {
+            CheckboxWithLabel(label = "60 days")
```

Comment:
I think these labels are going to be static values, so its better we can keep them in resource file.

Reply:
done